--- easytag-2.4.3/data/easytag.appdata.xml.in	2024-02-12 01:09:38.076272427 +1100
+++ easytag.appdata.xml.in	2024-02-12 01:21:21.975097451 +1100
@@ -8,6 +8,7 @@
 <!-- Name and summary taken from the desktop file. -->
     <_name>EasyTAG</_name>
     <_summary>Edit audio file metadata</_summary>
+    <developer_name>The EasyTAG developers</developer_name>
     <description>
         <_p>View and edit tags for MP3, MP2, MP4/AAC, FLAC, Ogg Opus, Ogg
         Speex, Ogg Vorbis, MusePack, Monkey's Audio and WavPack files.</_p>
