diff --color -ura easytag-2.4.3/data/easytag.appdata.xml.in easytag-2.4.3-patched/data/easytag.appdata.xml.in
--- easytag-2.4.3/data/easytag.appdata.xml.in	2016-08-14 22:44:13.000000000 +1000
+++ easytag-2.4.3-patched/data/easytag.appdata.xml.in	2024-11-01 12:58:12.424355009 +1100
@@ -8,6 +8,7 @@
 <!-- Name and summary taken from the desktop file. -->
     <_name>EasyTAG</_name>
     <_summary>Edit audio file metadata</_summary>
+    <developer_name>The EasyTAG developers</developer_name>
     <description>
         <_p>View and edit tags for MP3, MP2, MP4/AAC, FLAC, Ogg Opus, Ogg
         Speex, Ogg Vorbis, MusePack, Monkey's Audio and WavPack files.</_p>
@@ -19,15 +20,21 @@
     <url type="homepage">https://wiki.gnome.org/Apps/EasyTAG</url>
     <url type="bugtracker">https://bugzilla.gnome.org/enter_bug.cgi?product=easytag</url>
     <url type="help">https://help.gnome.org/users/easytag/stable/</url>
+    <launchable type="desktop-id">org.gnome.EasyTAG.desktop</launchable>
     <screenshots>
         <screenshot type="default">
-            <image>http://amigadave.com/temp/easytag-default.png</image>
+            <image>https://raw.githubusercontent.com/flathub/org.gnome.EasyTAG/master/screenshots/default.png</image>
+            <caption>Screenshot of the application</caption>
         </screenshot>
     </screenshots>
     <kudos>
         <kudo>AppMenu</kudo>
         <kudo>ModernToolkit</kudo>
     </kudos>
+    <content_rating type="oars-1.1"/>
+    <releases>
+        <release version="2.4.3" date="2016-12-05"/>
+    </releases>
     <update_contact>amigadave@amigadave.com</update_contact>
     <translation type="gettext">easytag</translation>
 </component>
