--- easytag-2.4.3/data/easytag.appdata.xml.in	2016-08-14 22:44:13.000000000 +1000
+++ easytag.appdata.xml.in	2021-11-05 11:42:27.272914510 +1100
@@ -21,7 +21,7 @@
     <url type="help">https://help.gnome.org/users/easytag/stable/</url>
     <screenshots>
         <screenshot type="default">
-            <image>http://amigadave.com/temp/easytag-default.png</image>
+            <image>https://raw.githubusercontent.com/flathub/org.gnome.EasyTAG/master/screenshots/default.png</image>
         </screenshot>
     </screenshots>
     <kudos>
